manifests/charts/kube-bench/values.yaml

```diff
@@ -39,7 +39,7 @@ pod:
 
 image:
   repository: m9sweeper/kube-bench
-  tag: latest
+  tag: 0.6.11
```

Comment:
I also want to see if perhaps we can stop having our own custom kube-bench image. Touch base with Grant on whether we could change the entrypoints in our kube-bench helm chart to install curl/wget or whatever

Reply:
https://github.com/m9sweeper/m9sweeper/blob/main/manifests/addons/kube-bench/Dockerfile

So we'd have to change the entrypoints to be prefixed with...

apk update && apk add curl &&

Reply:
Then we could remove https://github.com/m9sweeper/m9sweeper/blob/main/manifests/addons/kube-bench/Dockerfile

Reply:
https://github.com/m9sweeper/m9sweeper/blob/main/manifests/charts/kube-bench/templates/cronjob-ack.yaml#L62 - you can see how each job in the templates right now runs kube-bench in different ways. Instead of having our own image in order to install curl, we could just have it install curl when it boots up.

Reply:
https://hub.docker.com/r/aquasec/kube-bench/tags
Need the v